src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1006,17 +1006,17 @@ internal enum ErrorCode
         ERR_BadDynamicQuery = 1979,
         ERR_DynamicAttributeMissing = 1980,
         WRN_IsDynamicIsConfusing = 1981,
-        ERR_DynamicNotAllowedInAttribute = 1982,                    // Replaced by ERR_BadAttributeParamType in Roslyn.
+        //ERR_DynamicNotAllowedInAttribute = 1982,                    // Replaced by ERR_BadAttributeParamType in Roslyn.
         ERR_BadAsyncReturn = 1983,
         ERR_BadAwaitInFinally = 1984,
         ERR_BadAwaitInCatch = 1985,
         ERR_BadAwaitArg = 1986,
         ERR_BadAsyncArgType = 1988,
         ERR_BadAsyncExpressionTree = 1989,
-        ERR_WindowsRuntimeTypesMissing = 1990,
+        //ERR_WindowsRuntimeTypesMissing = 1990, //unused
         ERR_MixingWinRTEventWithRegular = 1991,
         ERR_BadAwaitWithoutAsync = 1992,
-        ERR_MissingAsyncTypes = 1993,
+        //ERR_MissingAsyncTypes = 1993, //unused
```

Comment:
I would prefer to say "unused in Roslyn" so that we know not to recycle this error id. (See, for example, [here](http://stackoverflow.com/questions/9600282/msbuild-doesnt-find-async-required-references))